docs/cli.md

```diff
@@ -58,6 +58,19 @@ ALIAS:
    scs-restart
 ```
 
+## `cf scs-restage`
+
+```
+NAME:
+   spring-cloud-service-restage - Restage a Spring Cloud Services service instance
+
+USAGE:
+      cf scs-restage SERVICE_INSTANCE_NAME
+
+ALIAS:
+   scs-restage
```

Comment:
Minor thing: I assume that actual aliases are going to be assigned for these new commands sometime soon?

Reply:
Well, they are : 
```
{
  Name:     "spring-cloud-service-start",
  HelpText: "Start a Spring Cloud Services service instance",
  Alias:    "scs-start",
  UsageDetails: plugin.Usage{
    Usage: "   cf scs-start SERVICE_INSTANCE_NAME",
  },
},
```